packages/access-api/test/helpers/ucanto-test-utils.js

```diff
@@ -92,6 +135,42 @@ export async function spaceRegistrationInvocation(
   return redeem
 }
 
+/**
+ * get an access-api invocation that will register an account.
+ * This is useful e.g. because some functionality (e.g. access/delegate)
+ * will fail unless the space is registered.
+ *
+ * @param {Ucanto.Signer<Ucanto.DID>} service - issues voucher/redeem. e.g. could be the same signer as access-api env.PRIVATE_KEY
+ * @param {Ucanto.DID<'mailto'>} account
+ * @param {Ucanto.DID<'key'>} agent
+ * @param {Ucanto.Principal} audience - audience of the invocation. often is same as issuer
+ * @param {number} lifetimeInSeconds
+ */
+export async function accountRegistrationInvocation(
+  service,
+  account,
+  agent,
+  audience = service,
+  lifetimeInSeconds = 60 * 15
+) {
+  const registerSpace = await Access.confirm
+    .invoke({
+      issuer: service,
+      audience,
+      with: service.did(),
+      lifetimeInSeconds,
+      // We link to the authorization request so that this attestation can
```

Comment:
>  We link to the authorization request

what's this referring to? I see the issuer set to be the account, the audience to the agent, and the attestations set to full open, but can't figure out in what sense this is a "link to the authorization request"

Reply:
I don't think it makes sense either. It's from copypasta. Good catch. i removed it